### PR TITLE
[RFR] - Added security rights on GroupController

### PR DESCRIPTION
**Pull request by mickaelandrieu**
* Add security annotations on GroupController
* Rest tests are now executed as sudoer
* Rest tests can now be executed with this command ``phpunit --group=Rest``

### CODE REVIEW

File: Rest/Controller/GroupController.php [hunk 1]
```diff
@@ -109,6 +113,7 @@ public function putAction(Group $group, Request $request)
      *  @Assert\NotBlank(message="Name is required"),
      *  @Assert\Length(max=50, minMessage="Maximum length of name is 50 characters")
      * })
+     * @Rest\Security("is_fully_authenticated() & has_role('ROLE_API_USER')")
```
Comment (eric-chau) [inline Rest/Controller/GroupController.php]: you should test if user can create a group no?

**Comment by eric-chau** [comment]
:+1:, thanks for your contribution @mickaelandrieu

File: Rest/Controller/GroupController.php (hunk 1 above)
Comment (mickaelandrieu) [inline Rest/Controller/GroupController.php]: You're right, I will update the expression.

**Comment by mickaelandrieu** [comment]
thank you for review @eric-chau

**Pull request merged**